instagram/bind.py

```diff
@@ -94,10 +94,16 @@ def _build_path(self):
             for variable in re_path_template.findall(self.path):
                 name = variable.strip('{}')
 
-                try:
-                    value = quote(self.parameters[name])
-                except KeyError:
-                    raise Exception('No parameter value found for path variable: %s' % name)
+                if name == 'tag_name':
+                    # XXX: tags can have unicode (non-ascii) characters, but in order
+                    # to do the signature computation correctly, we must not urlencode
+                    # them.
+                    value = self.parameters[name]
```

Comment:
Note that httplib2 already converts uri's to iri's under the covers, so the `quote()` function is largely unneeded. This will break if tags include invalid url characters though, such as ?, {}, & etc but I'm fairly certain those are not allowed anyhow. They don't appear to work on the tireswing client either.